### ISSUE TITLE
'highland' object is no longer added to the global scope in the browser.

### ISSUE THREAD

**Issue by dayoadeyemi** [caolan/highland#389]
adding `<script src="https://cdn.rawgit.com/caolan/highland/master/dist/highland.js" type="text/javascript"></script>` doesn't seem to add the highand object to the global scope any more, where as `<script src="https://cdn.rawgit.com/caolan/highland/2.5.1/dist/highland.js" type="text/javascript"></script>` does

**Comment by quarterto** [comment]
That's possibly related to our recent upgrade of Browserify. I'll take a look.

**Comment by svozza** [comment]
Should there be a minified version of the code in `dist` too?

**Comment by svozza** [comment]
Hmmm, slight problem. How can I run the unit tests against the minified code? If I try to run them against `dist/highland.js` on its own - never mind the minified version of that file - they fail.

**Comment by vqvu** [comment]
Oh, yes. Adding a minified highland.js would be great.

Running browser tests: https://github.com/caolan/highland/blob/master/Contributing.md#running-the-tests

I don't see any errors. You have to make sure to rebuild `test/bundle.js` though. The version that comes with the repo is never updated.

**Comment by svozza** [comment]
Yeah, that runs fine but technically we're not testing `dist/highland.js` with those tests (there's a TODO in `browser.html` about fixing this). I tried to get something working using `rewireify` but no luck. I guess it doesn't matter that much because Browserifying the unit tests is close enough in the `dist/highland.js` case but I'd be less confident about the minified version of the code.
